containers/app/Dockerfile

```diff
@@ -45,6 +45,7 @@ RUN apt-get update -y \
     && apt-get install -y curl ssh sudo
 
 RUN sed -i 's/^UID_MIN.*/UID_MIN 499/' /etc/login.defs # Default is 1000, but OSX is often 501
+RUN sed -i 's/^UID_MAX.*/UID_MAX 1000000/' /etc/login.defs # Default is 60000, but we've seen up to 200000
```

Comment:
I know `UID_MAX` helps to distinguish between ordinary users and system users. But can you explain more about why change the default value?